sql/planner/operation_string.go

```diff
@@ -0,0 +1,33 @@
+// Code generated by "stringer -type=Operation"; DO NOT EDIT.
```

Comment:
What's the purpose of this file?

Reply:
It was generated by this tool https://godoc.org/golang.org/x/tools/cmd/stringer, it's a way to generate `String() string` methods to enums